pkg/subctl/cmd/export.go

```diff
@@ -0,0 +1,90 @@
+/*
+© 2020 Red Hat, Inc. and others.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/submariner-io/lighthouse/pkg/apis/lighthouse.submariner.io/v2alpha1"
+	lighthouseClientset "github.com/submariner-io/lighthouse/pkg/client/clientset/versioned"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	exportCmd = &cobra.Command{
+		Use:   "export",
+		Short: "Exports a service",
+		Long:  "This command simplifies exporting service(s) to other clusters",
+	}
+	exportServiceCmd = &cobra.Command{
+		Use:   "service",
+		Short: "Creates a ServiceExport resource",
+		Long:  "This command creates a ServiceExport resource with a given name which causes the Service of the same name to be accessible to other clusters",
+		Run:   exportService,
+	}
+	serviceNamespace string
+)
+
+func init() {
+	addKubeconfigFlag(exportCmd)
+	addServiceExportFlags(exportServiceCmd)
+	exportCmd.AddCommand(exportServiceCmd)
+	rootCmd.AddCommand(exportCmd)
+}
+
+func addServiceExportFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(&serviceNamespace, "namespace", "default", "Namespace in which service to be exported belongs")
```

Comment:
I would expect that we could grab the current namespace in the kubeconfig. So it behaves like kubectl in this, otherwise it would be confusing.

stackoverflow to the rescue of us: https://stackoverflow.com/questions/55314152/how-to-get-namespace-from-current-context-set-in-kube-config ':D


```
contexts:
- context:
    cluster: default
    namespace: pr-brancher-webhook
    user: default
  name: default
```

and then...


```
current-context: kind-cluster3
```

Reply:
done.